app/views/associate_ucr_confirmation.scala.html

```diff
@@ -14,33 +14,33 @@
  * limitations under the License.
  *@
 
-@import controllers.routes
-@import uk.gov.hmrc.play.views.html._
-@import controllers.storage.FlashKeys
 @import views.Title
-@import views.html.templates.main_template
+@import components.gds.gds_main_template
+@import components.gds.pageTitle
+@import uk.gov.hmrc.govukfrontend.views.html.components.govukInsetText
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.insettext.InsetText
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+@import views.html.components.gds.link
 
-@this(main_template: main_template)
+@this(
+        govukLayout: gds_main_template,
+        pageTitle: pageTitle,
+        govukInsetText: govukInsetText
+)
 
-@(associateUcrKind: String, ucr: String)(implicit request: Request[_], messages: Messages)
+@()(implicit request: Request[_], messages: Messages)
 
-@prefix = @{ s"associate.$associateUcrKind.confirmation" }
-@associateLink = {<a href="@routes.ChoiceController.startSpecificJourney(forms.Choice.AssociateUCR)">@messages("association.confirmation.associateOrDepart.associate")</a>}
-@shutMucrLink = {<a href="@routes.ChoiceController.startSpecificJourney(forms.Choice.Departure)">@messages("association.confirmation.associateOrDepart.depart")</a>}
+@gotoTimelineLink = {<a class="govuk-link" href="@routes.ViewSubmissionsController.displayPage()">@messages("movement.confirmation.notification.timeline.link")</a>}
 
-@main_template(title = Title(s"$prefix.tab.heading")) {
+@govukLayout(title = Title(s"movement.confirmation.title.ASSOCIATE_UCR")) {
 
-    @components.highlight_box_with_reference(
-        headingText = messages(s"$prefix.heading", ucr)
-    )
+    @pageTitle(text = messages(s"movement.confirmation.title.ASSOCIATE_UCR"), classes = "govuk-heading-xl")
 
-    @components.confirmation_status_info()
+    @govukInsetText(InsetText(
+        content = HtmlContent(messages("movement.confirmation.header")+"<br>"+messages("movement.confirmation.header.check", gotoTimelineLink))
+    ))
 
-    <h2 id="what-next">@messages("movement.confirmation.whatNext")</h2>
-
-    <div id="next-steps">
-        @Html(messages("association.confirmation.associateOrDepart", associateLink, shutMucrLink))
-    </div>
-
-    @components.button_link("site.backToStart", routes.ChoiceController.displayPage())
-}
\ No newline at end of file
+    <p class="govuk-body">
+    @link(message = messages("movement.confirmation.redirect.link"), href = routes.ChoiceController.displayPage())
```

Comment:
Does this message key exist in feature branch from previous PR?

Reply:
Yes